### ISSUE TITLE
Unable to complete cargo build release

### ISSUE THREAD

**Issue by chrizteroo** [hyperledger/indy-sdk#2588]
Am unable to complete cargo build, it was stock in 275/276.
using AWS EC2 ubuntu 22.04 jammy
![image](https://user-images.githubusercontent.com/108843788/224515016-409c7c0b-6fcb-4dc8-b50b-f0c7790c8d14.png)

**Comment by zhb001** [comment]
Their document describes the use of  ubuntu16.4 or 18.4, but I did not compile it successfully, but directly used the libindy provided by them

**Comment by chrizteroo** [comment]
@zhb001 what is the solution to resolve this? you mean I should re-image my ubuntu 16.4 or 18.4? 
kindly provide the link to the libindy you mention above.
Thanks.

**Comment by zhb001** [comment]
https://github.com/hyperledger/indy-sdk#ubuntu-based-distributions-ubuntu-1604-and-1804

**Comment by chrizteroo** [comment]
Cargo build release issue identified.
Noticed that my memory and ram were small when I installed ubuntu on it, I also noticed my mac os wireless transfer and download rate was an issue. I rectified all by increasing them, and everything was fine.
Lastly remember it was stuck on 275 of 276. Before starting again I ran the followings:
1. whereis OpenSSL
2. sudo apt-get remove OpenSSL
3. sudo apt-get install OpenSSL (Note: this is for ubuntu environment)
4. cargo update
5. cargo build --release
That is all.
